module/data/item/templates/activated-effect.mjs

```diff
@@ -336,7 +336,7 @@ export default class ActivatedEffectTemplate extends SystemDataModel {
   get activatedEffectChatProperties() {
     foundry.utils.logCompatibilityWarning("ActivatedEffectTemplate#activatedEffectChatProperties is deprecated. "
       + "Please use ActivatedEffectTemplate#activatedEffectCardProperties.",
-      { since: "DnD5e 3.0", until: "DnD5e 3.2", once: true });
+    { since: "DnD5e 3.0", until: "DnD5e 3.2", once: true });
```

Comment:
I know the linter says this is correct, but it doesn't feel like the correct indentation. I'd say we just move the first line of the warning down and have everything indented:

```javascript
    foundry.utils.logCompatibilityWarning(
      "ActivatedEffectTemplate#activatedEffectChatProperties is deprecated. "
      + "Please use ActivatedEffectTemplate#activatedEffectCardProperties.",
      { since: "DnD5e 3.0", until: "DnD5e 3.2", once: true });
```

Reply:
Yeah, sounds reasonable